content/guides/getstarted/intro-to-ai.md

```diff
@@ -0,0 +1,544 @@
+   - Examples: Eliza, LangChain, AutoGPT, Vercel AI
+
+3. **Domain-Specific Agent Kits (like Solana Agent Kit)**
+   - Built for specific use cases (e.g., blockchain operations)
+   - Provide pre-built tools and integrations
+   - Abstract away technical complexity
+   - Ready-to-use blockchain functions
+   - Examples: Solana Agent Kit, GOAT Toolkit
+
+Think of it like this:
+
+- LLMs are the "brain" that can understand and generate language
+- Agent frameworks are the "operating system" that manages the AI's behavior
+- Domain-specific kits are the "apps" that give the AI specific capabilities
+
+# Building AI Agents for Solana
+
+This guide will show you how to:
+
+- Build a basic AI agent using SendAI's Solana Agent Kit
+- Create a Twitter bot with Eliza Framework
+- Understand on-chain AI integration options
+- Options on how to build advanced trading bots
+
+## Introduction to AI Agents on Solana
+
+AI agents on Solana are autonomous programs that can interact with the
+blockchain using natural language processing and machine learning. They can:
+
+- Execute transactions
+- Monitor blockchain activity
+- Interact with smart contracts
+- Provide user assistance
+- Automate complex workflows
+
+## Available Frameworks
+
+Several frameworks are available for building AI agents on Solana:
+
+1. **SendAI's Solana Agent Kit**
+
+- A collection of tools for building agents on Solana
+- Compatible with Eliza, Langchain, and Vercel AI SDK
+- Built-in Solana tools and utilities
+- Natural language processing capabilities
+- [Solana Agent Kit](https://github.com/sendaifun/solana-agent-kit)
+- [Documentation](https://docs.solanaagentkit.xyz/)
+
+2. **Eliza Framework**
+
+- Combines datalayer, llm and agents and has a dedicated Solana plugin
+- Extensible agent architecture
+- Easy integration with existing projects and can be used for example for
+  Twitter (X), telegram or discord bots
+- [GitHub Repository](https://github.com/elizaOS/eliza)
+
+3. **GOAT Toolkit**
+
+- Comprehensive development toolkit
+- Cross-chain compatibility
+- [GitHub Repository](https://github.com/goat-sdk/goat)
+
+4. **Rig Framework**
+
+- Native Rust implementation
+- Good for high performance trading bots
+- Can be combined with the Listen.rs toolkit for Solana integration transaction
+  monitoring and sending transactions using Jito bundles for example
+- [GitHub Repository](https://github.com/0xPlaygrounds/rig)
+- [Listen.rs Repository](https://github.com/piotrostr/listen)
+
+5.
+
+- If you are building an AI framework feel free to open a PR to add your
+  framework to the list
+
+## Getting Started with SendAI
+
+Let's start by setting up a basic AI agent using SendAI's Solana Agent Kit that
+can perform actions on the Solana blockchain and that we can chat with:
+
+<Steps>
+
+### Install dependencies
+
+You will need to have [node](https://nodejs.org/en/download/) with version
+`23.x.x` installed. Open an empty folder using vscode or cursor and run the
```

Comment:
Add a callout that you must use node 23.x.x, otherwise the agents will throw errors.

Reply:
Done